### ISSUE TITLE
Ability to choose storage type

### ISSUE THREAD

**Issue by amellini** [marcuswestin/store.js#276]
I'd like to choose the default storage engine (sessionStorage, i need to store different information for different tabs), i don't use NPM.
It will be a good solution to allow the user to select a "default" storage and if not available to fall to other engine.

There's a simple way to do this or i need to create my own store pack (with the list of all engines)?

**Comment by yonib05** [comment]
+1

**Comment by galtenberg** [comment]
Now I get why the docs don't explain how to do this. Would love to! I currently import via unpkg. 

🙏

**Comment by dgbeck** [comment]
yes, sometimes you want localStorage and sometimes you want sessionStorage, for different use cases. but you want to use the same nice API for both...

**Comment by TafkaMax** [comment]
If i am using NPM how do I choose storage type, I also want to use sessionStorage.

**Comment by roy-rayyone** [comment]
try this

```
import store from 'store';
// @ts-ignore
import cookieStorage from 'store/storages/cookieStorage';

const cs = store.createStore([cookieStorage]);
cs.set(key, value);
```
